### PR TITLE
Updating/adding pages that will appear in the header 

### PR DESCRIPTION
**Pull request by Jen-OMalley**
Updating/adding pages that will appear in the header for the 2021 September Training website

### CODE REVIEW

File: _config.yaml [hunk 1]
```diff
@@ -35,7 +35,11 @@ header_pages:
   # as described at https://github.com/AlexsLemonade/training-specific-template#customizing-the-new-repository-for-an-individual-training-workshop
   - workshop/SCHEDULE.md
   - workshop/workshop-structure.md
+  - workshop/software-setup.md
   - workshop/workshop-materials.md
+  - workshop/resources-for-consultation-sessions.md
+  - workshop/posting-errors-guidelines.md
+  - workshop/working-with-your-own-data.md
```
Comment (jashapiro) [inline _config.yaml:42]: This page is in a slightly different place from the others for whatever reason. I see that you changed the `nav_title` in a separate PR, but we should put that change in the same branch/PR as this one.
```suggestion
  - working-with-your-data/working-with-your-own-data.md
```

Comment (Jen-OMalley) [reply _config.yaml]: Okay **@jashapiro**! I attempted to move that change to this branch, but not totally sure if I did that right?

Comment (jashapiro) [reply _config.yaml]: Looks good! I think you can close the https://github.com/AlexsLemonade/2021-september-training/pull/19 PR now

Comment (Jen-OMalley) [reply _config.yaml]: Done, thanks!

File: _config.yaml [hunk 2]
```diff
@@ -35,7 +35,11 @@ header_pages:
   # as described at https://github.com/AlexsLemonade/training-specific-template#customizing-the-new-repository-for-an-individual-training-workshop
   - workshop/SCHEDULE.md
   - workshop/workshop-structure.md
+  - workshop/software-setup.md
   - workshop/workshop-materials.md
+  - workshop/resources-for-consultation-sessions.md
```
Comment (jashapiro) [inline _config.yaml:40]: Can we change the `nav_title` on this page too? Right now it is "Consultation Sessions", but I think "Resources" might be a better title. Others may disagree!

Comment (Jen-OMalley) [reply _config.yaml:40]: Sounds good. Also done!

File: _config.yaml [hunk 3]
```diff
@@ -35,7 +35,11 @@ header_pages:
...
+  - workshop/posting-errors-guidelines.md
+  - working-with-your-data/working-with-your-own-data.md
```
Comment (jashapiro) [inline _config.yaml:42]: Trying to think about whether we should adjust the order here. Also, I think we can probably get away with not having the software setup in the nav bar. People hopefully only need that once, and it is linked from the home page.
```suggestion
  - workshop/workshop-materials.md
  - workshop/resources-for-consultation-sessions.md
  - working-with-your-data/working-with-your-own-data.md
  - workshop/posting-errors-guidelines.md
```

File: working-with-your-data/working-with-your-own-data.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Working with your own data
+nav_title: Work with your Data
```
Comment (jashapiro) [inline working-with-your-data/working-with-your-own-data.md:3]: I'm trying to come up with something a bit shorter here, but having trouble. 
"Data management"?
"Using your own data"? (that's the same length.)
"Managing Data"?

**Review by jashapiro — COMMENTED** [review]
This looks good, but I'd like to have other opinions about some of my suggestions!

File: working-with-your-data/working-with-your-own-data.md (hunk 4 above)
Comment (jaclyn-taroni) [reply working-with-your-data/working-with-your-own-data.md:3]: I haven't looked at this locally to see how it looks, but as far as indicating what the content is I feel that "Work with your data" or "Using your own data" is about right. We could do "Your Data" but that seems a little weird?

Comment (jashapiro) [reply working-with-your-data/working-with-your-own-data.md:3]: Okay, How about this? 
```suggestion
nav_title: Using Your Data
```
If not, we should switch to a (modified) title case for better emphasis.
```suggestion
nav_title: Work with Your Data
```

Comment (Jen-OMalley) [reply working-with-your-data/working-with-your-own-data.md:3]: I think I prefer "Using Your Data." Looks neater to me for some reason.

Comment (jashapiro) [reply working-with-your-data/working-with-your-own-data.md:3]: Okay, then you can just click the "Commit suggestion" button to make that change!

**Review by jashapiro — APPROVED** [review]
LGTM!

**Pull request merged**